### PR TITLE
290931: Define resource quotas with custom or default values and Limit Range with initial set of values.

### PR DESCRIPTION
**Pull request by ajaysoni-cap**
# **What this PR does / why we need it**:
Define resource quotas with custom or default values and Limit Range with initial set of values.
Relates to ADO Work Item [AB#290931](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/290931)

# **Special notes for your reviewer**
N/A

# Testing
N/A

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
LethalCompanyQuotaGIF

### CODE REVIEW

**Review by DaRoszko — APPROVED** [review]

File: services/ffc/ffc-demo/base/patch/kustomize.yaml
Comment (aadev1) [inline services/ffc/ffc-demo/base/patch/kustomize.yaml:1]: @ajaysoni-cap There's a chance these could get overwritten if someone regenerates these file using the flux manifest automation scaffolding.

File: services/base/resource-quota.yaml
Comment (aadev1) [inline services/base/resource-quota.yaml:1]: @ajaysoni-cap Does the path to resource-quota.yaml need to be included in the Kustomization.yaml

Comment (ajaysoni-cap) [reply services/base/resource-quota.yaml:1]: Good spot.. Included the file in base kustomization.

File: services/ffc/ffc-demo/base/patch/kustomize.yaml
Comment (ajaysoni-cap) [reply services/ffc/ffc-demo/base/patch/kustomize.yaml:1]: We need to have a look at the scaffolder for these values.. may need to discuss as it should not overwrite the values..

Comment (aadev1) [reply services/ffc/ffc-demo/base/patch/kustomize.yaml:1]: I don't think it's an issue when we Onboard new services.  It's when someone updates something that needs to be reflected in all services.  Currently the scaffolder scripts are being used to regenerate all the files with any platform  additions and raising a PR e.g. The recent changes Ganesh made for the storage account private endpoints.  Thought it was worth mentioning and yes I think we need to discuss.

Comment (ajaysoni-cap) [reply services/ffc/ffc-demo/base/patch/kustomize.yaml:1]: That's a valid point but I think it's ok even if the scaffolder overwrites the values for now in the demo service namespace...
We'll discuss this as part of the scaffolder integration work.

**Review by DaRoszko — APPROVED** [review]

**Review by aadev1 — APPROVED** [review]

**Pull request merged**